database/src/retriable.rs

```diff
@@ -23,7 +23,7 @@ macro_rules! await_retry_or_panic {
                         })?
 
                         tracing::error!(
-                             target: crate::INDEXER_FOR_EXPLORER,
+                             target: crate::EXPLORER_DATABASE,
```

Comment:
nit:
All spell checkers always insist on `retryable`
Do we want to change the file name and the function name by any chance? :)

Reply:
shame on me, we definitely should rename it if it is incorrect. @morgsmccauley could you please?

Reply:
Done